src/bp/core/services/auth/auth-service.ts

```diff
@@ -44,11 +44,12 @@ export default class AuthService {
     return await this.findUser({ email }, selectFields)
   }
 
-  async checkUserAuth(email: string, password: string, newPassword?: string) {
+  async checkUserAuth(email: string, password: string, newPassword?: string, ipAddress: string = '') {
```

Comment:
Can it really happen that we have no IP address? If thats the case I'd just add "unknown" or remove the `from IP` part of the log.